src/Util.cc

```diff
@@ -579,6 +582,80 @@ std::optional<math::Vector3d> sphericalCoordinates(Entity _entity,
   // Return degrees
   return math::Vector3d(IGN_RTOD(rad.X()), IGN_RTOD(rad.Y()), rad.Z());
 }
+
+//////////////////////////////////////////////////
+// Getting the first .sdf file in the path
+std::string findFuelResourceSdf(const std::string &_path)
+{
+  if (!common::exists(_path))
+    return "";
+
+  for (common::DirIter file(_path); file != common::DirIter(); ++file)
+  {
+    std::string current(*file);
+    if (!common::isFile(current))
+      continue;
+
+    auto fileName = common::basename(current);
+    auto fileExtensionIndex = fileName.rfind(".");
+    auto fileExtension = fileName.substr(fileExtensionIndex + 1);
+
+    if (fileExtension == "sdf")
+    {
+      return current;
+    }
+  }
+  return "";
+}
+
+//////////////////////////////////////////////////
+std::string IGNITION_GAZEBO_VISIBLE resolveSdfWorldFile(
+    const std::string &_sdfFile, const std::string &_fuelResourceCache)
```

Comment:
I think this is tripping up the linter

```suggestion
std::string resolveSdfWorldFile(
    const std::string &_sdfFile, const std::string &_fuelResourceCache)
```

Reply:
Fixed in [7d33cf0](https://github.com/ignitionrobotics/ign-gazebo/pull/1333/commits/7d33cf0aafacefa3a4bdaa1522ce02b7d7b5263b)